prefork/prefork_test.go

```diff
@@ -0,0 +1,206 @@
+package prefork
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"reflect"
+	"runtime"
+	"testing"
+
+	"github.com/valyala/fasthttp"
+)
+
+func init() { //nolint:gochecknoinits
+	setUp()
```

Comment:
Please move this into the tests. If I'm looking at `Test_ListenAndServe` for example, it is unclear to me that this is supposed to be a child process and not the master process. It's always a bad idea to do things in `init` in tests.

Reply:
Done!